pkg/apiserver/api_installer.go

```diff
@@ -139,7 +139,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	// we're trying to register here
 	fqKindToRegister := unversioned.GroupVersionKind{}
 	for _, fqKind := range fqKinds {
-		if fqKind.Group == a.group.GroupVersion.Group {
+		if fqKind.Group == a.group.GroupVersion.Group || (fqKind.Group == extensions.GroupName && fqKind.Kind == "ScaleTwo") {
```

Comment:
I either want to allow the RESTStorage the option of saying, "I'll take care of interpreting those bytes" or "convert to X at all cost".  I don't like the patch we carry that allows configured overrides for a specific subresource.